clang/lib/Sema/SemaTemplate.cpp

```diff
@@ -6507,8 +6508,13 @@ bool Sema::CheckTemplateArgumentList(
       bool PackExpansionIntoNonPack =
           NewArgs[ArgIdx].getArgument().isPackExpansion() &&
           (!(*Param)->isTemplateParameterPack() || getExpandedPackSize(*Param));
-      if (PackExpansionIntoNonPack && (isa<TypeAliasTemplateDecl>(Template) ||
-                                       isa<ConceptDecl>(Template))) {
+      // Core issue 1430: Don't diagnose this pack expansion when partial
```

Comment:
```suggestion
      // Core issue CWG1430: Don't diagnose this pack expansion when partial
```

(makes it easier to grep for)